README.md

```diff
@@ -5,24 +5,31 @@ Create, fork and edit d3.js code snippets for use with bl.ocks.org right in the
 Eventually we will host this with a domain, but for now let's just get it running locally by following the instructions in Development.
 
 # Development
-
-MongoDB and Redis are required (for sessions and users), they are currently configured inside `server.js`
+MongoDB and Redis are required (for sessions and users) so make sure you install them before proceeding any further. If you would like to tweak the default parameters, you can do so my modifying `server.js`.
 
 ```bash
 git clone https://github.com/enjalot/building-blocks.git
 cd building-blocks/
 npm install
-# run webpack to build the js & css bundles
+# run webpack to build the js & css bundles (note that this does not terminate, so feel free to kill this when no apparent progress is being made)
 npm run local
-# run the server (use node-dev server.js for auto-restarting on changes to server code)
-node server.js
+# now set up mongodb with the following commands
+mkdir -p /data/db
+mongod
+# now set up redis with the following commands
+redis-server
+# building-blocks expects something in the secrets.json file, so the last step is: (to place replace this with your credentials, see Registering Github Credentials below)
+mv secrets.json-example secrets.json
+# run the server (use nodemon server.js for auto-restarting on changes to server code). see http://nodemon.io for information on installing nodemon.
+nodemon server.js
```

Comment:
i prefer node-dev, but I think the default instructions shouldn't require people to install something different. lets leave it as node server.js and use the comments to mention nodemon or node-dev (which can just be npm installed)